sceptre/synapseprod/templates/accounts.yaml

```diff
@@ -1,247 +0,0 @@
-Description: Setup IAM policies, groups and accounts
-AWSTemplateFormatVersion: 2010-09-09
-Parameters:
-  InitNewUserPassword:
-    Type: String
-    NoEcho: true
-Resources:
-  # !! IMPORTANT !! - AWS API will refuse to remove users that have attached resources.
-  # Therefore you must do the following before deleting them from this file:
-  # 1. Detach or remove the following user resources: login profile, attached
-  #    MFA device, access-keys, ssh-keys, and policies.
-  # 2. Detach the user from all groups.
-  AWSIAMXavierSchildwachterUser:
-    Type: 'AWS::IAM::User'
-    Properties:
-      UserName: x.schildwachter@sagebase.org
-      Groups:
-        - !Ref AWSIAMAllUsersGroup
-        - !Ref AWSIAMDeveloperUsersGroup
-        - !Ref AWSIAMBillingManagersGroup
-      LoginProfile:
-        Password: !Ref InitNewUserPassword
-        PasswordResetRequired: true
-  AWSIAMDeveloperUsersGroup:
-    Type: 'AWS::IAM::Group'
-    Properties:
-      ManagedPolicyArns:
-        - !Ref AWSIAMDevelopersAdditionalAccessPolicy
-  AWSIAMAnalyticsUsersGroup:
-    Type: 'AWS::IAM::Group'
-    Properties:
-      ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/AmazonAthenaFullAccess
-        - arn:aws:iam::aws:policy/AWSGlueConsoleFullAccess
-        - !Ref AWSIAMAnalyticsUsersAccessPolicy
-  AWSIAMWebEngineerUsersGroup:
-    Type: 'AWS::IAM::Group'
-    Properties:
-      ManagedPolicyArns:
-        - !Ref AWSIAMStaticSynapseUsersAccessPolicy
-  AWSIAMAdminRole:
```

Comment:
you can't remove this until you've removed these dependencies..

```
sceptre/synapsedev/templates/SynapseCMK-template.json:                                        { "Fn::ImportValue": "us-east-1-accounts-AWSIAMAdminRoleArn" },

sceptre/synapseprod/templates/SynapseCMK-template.json:                                        { "Fn::ImportValue": "us-east-1-accounts-AWSIAMAdminRoleArn" },
```

Reply:
Thanks! Removed references to the role in #999 .